about.html

```diff
@@ -0,0 +1,38 @@
+<!DOCTYPE html> 
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title> About Alaere </title>
+    <link rel="stylesheet" href="styles/style.css">
+</head>
+<body>
+  <nav>
+      <ul class="nav-list">
+        <li><a href="http://127.0.0.1:5500/index.html"> Home </a></li>
```

Comment:
When linking within a site, don't include the server part of the address `http://127.0.0.1:5500/`. Only include this when a link leaves your site. Notice that because of the hard-coded 127.0.0.1, the links are broken in the deployed site (since the address of the deployed site isn't 127.0.0.1).